README.md

```diff
@@ -29,9 +29,14 @@ Configure environment variables as necessary.
     export SSS_CLAIM_BIP39_SEED="bip39 wallet seed" // in-flight claims seed
     export SSS_CLAIM_LTCTESTNET_ADDRESS="ltctestnet addr" // Claim to address
     export SSS_CLAIM_TESTNET_ADDRESS="addr" // Override using LND for claims
-    export SSS_LND_GRPC_HOST="host:port" // LND GRPC API
-    export SSS_LND_MACAROON="base64 exported lnd macaroon file" // LND macaroon
-    export SSS_LND_TLS_CERT="base64 exported TLS cert" // LND TLS cert
+
+    export SSS_BTC_LND_GRPC_HOST="host:port" // LND GRPC API, LND daemon configured for BTC
```

Comment:
The convention following bitcoinjslib is to refer to networks by their network name:

BTC is BITCOIN
LTC is LITECOIN

BTC Testnet should be TESTNET not BTC
LTC Testnet should be LTCTESTNET not LTC